applications/ConvectionDiffusionApplication/custom_utilities/bfecc_convection.h

```diff
@@ -42,8 +42,10 @@ template<std::size_t TDim> class BFECCConvection
 public:
     KRATOS_CLASS_POINTER_DEFINITION(BFECCConvection<TDim>);
 
-    BFECCConvection(typename BinBasedFastPointLocator<TDim>::Pointer pSearchStructure)
-        : mpSearchStructure(pSearchStructure)
+    BFECCConvection(
+        typename BinBasedFastPointLocator<TDim>::Pointer pSearchStructure, 
+        const bool partial_dt = false)
```

Comment:
```suggestion
        const bool PartialDt = false)
```
(style guide convention)